### ISSUE TITLE
[Bug Report] Dropdown accessibility error: 'Invalid ARIA attribute value: aria-haspopup="list"'

### ISSUE THREAD

**Issue by k2gray** [ElemeFE/element#9365]
### Element UI version
2.0.11

### OS/Browsers version
Using aXe Accessibility Engine Chrome extension

### Vue version
2.5.13

### Reproduction Link
[https://jsfiddle.net/mmx38qxw/231/](https://jsfiddle.net/mmx38qxw/231/)

### Steps to reproduce
- Create any <el-dropdown> component
- Evaluate the html output using an accessibility checker. Should conform to WCAG 2.0 level AA. Using aXe chrome extension: http://bitly.com/aXe-Chrome 

### What is Expected?
The dropdown trigger element 'el-dropdown-link' should have an aria-haspopup value of 'listbox' or 'true'

### What is actually happening?
The dropdown trigger element 'el-dropdown-link' has an aria-haspopup value of 'list' (not a valid value for 'aria-haspopup')

**Comment by Leopoldthecoder** [comment]
@maranran Could you verify this?

**Comment by maranran** [comment]
yes，aria-haspopup's value should be 'listbox' not 'list' . thanks for your issue, it will be fixed later @Leopoldthecoder @k2gray

**Comment by Morgbn** [comment]
Hello,
I would like to reopen this issue because the pull request has not been approved,
in the current code "aria-haspopup" is still 'list' instead of 'listbox' ([here](https://github.com/ElemeFE/element/blob/dev/packages/dropdown/src/dropdown.vue#L197))
